README.md

```diff
@@ -138,7 +138,7 @@ Dashboards needs to be imported manually. Below is an example for importing Dash
 
 1. Download and configure [OCI CLI](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliinstall.htm) or open cloud-shell where OCI CLI is pre-installed. Alternative methods like REST API, SDK, Terraform etc can also be used.
 2. Find the **OCID** of the compartment, where the dashboards need to be imported.
-3. Download the dashboard JSONs from [here](logan/terraform/oke/modules/dashboards/dashboards_json/) (TBD).
+3. Download the dashboard JSONs from [here](terraform/modules/dashboards/dashboards_json/).
```

Comment:
Can we close https://github.com/oracle-quickstart/oci-kubernetes-monitoring/pull/39 with the new jsons?

Reply:
As per our discussion pulled in the change related to Query part only but not the visualisation.